client/machines.go

```diff
@@ -18,9 +18,14 @@ func (m *Machines) client() APIClient {
 }
 
 // Get fetches a list machines.
-func (m *Machines) Get() ([]entity.Machine, error) {
+func (m *Machines) Get(machinesParams *entity.MachinesParams) ([]entity.Machine, error) {
+	qsp, err := query.Values(machinesParams)
+	if err != nil {
+		return nil, err
```

Comment:
Can you return nil for a list? I thought you'd have to return `[]entity.Machine{}`

Reply:
Ah, guess you can.